### PR TITLE
fix(legacy-element-mixin): _log with single parameter #5006

### LINKED ISSUE #5006: [bug] _log with single parameter does not show properly
### Description
When `_log` call with single string parameter (like `this._log('test')`) it does not show properly on the console!
Because _logger convert `log('test')` to `log('t', 'e', 's', 't')`!

#### Live Demo

http://jsbin.com/luhaxab/1/edit?js,console

### PR DESCRIPTION
**Pull request by AliMD**
### Reference Issue

Fixes #5006
Supersedes and closes #4924

### CODE REVIEW

**Comment by AliMD** [comment]
@TimvdLippe If I want to make a test cant use `https://github.com/Polymer/polymer/blob/master/test/unit/logging.html`
Because it's about `_logger` itself and this fn replace with another fn for test
I force to make another test unit and I don't think its necessary 
I make a simple test with overwrite console.log for make you happy ;)

**Comment by AliMD** [comment]
@kevinpschaaf @azakus Could you please change the reviewer 😄

**Comment by AliMD** [comment]
@TimvdLippe Any order sir?

File: test/unit/logging.html [hunk 1]
```diff
@@ -85,6 +85,22 @@
       assert.includeMembers(output, args, `${args} should be in ${output}`);
     });
   });
+  suite('_logger', () => {
+    test('_log with single parameter', () => {
+      const msg = 'log test';
+      const logHistory = [];
+      const orgConsole = console;      
+      window.console = {
```
Comment (TimvdLippe) [inline test/unit/logging.html]: You can do the following with `sinon` (http://sinonjs.org/releases/v4.1.3/spies/):

```js
test('_log with single parameter', () => {
  const msg = 'log test';
  const spy = sinon.spy();
  const oldConsole = window.console;

  window.console = spy;
  Polymer.Base._log(msg);
  window.console = oldConsole;

  assert.isTrue(spy.calledOnce);
  assert.isTrue(spy.calledWith(msg));
});
```

Comment (AliMD) [reply test/unit/logging.html]: @TimvdLippe Any pros for this case?

Comment (TimvdLippe) [reply test/unit/logging.html]: For consistency, as we use it throughout our testing suite: https://github.com/Polymer/polymer/search?utf8=%E2%9C%93&q=sinon.spy&type=

**Comment by AliMD** [comment]
@TimvdLippe Should I close this PR?
Any hope to merge?

**Comment by TimvdLippe** [comment]
Please process the feedback of https://github.com/Polymer/polymer/pull/5007#discussion_r159123572 and then this is good to be merged :smile:

**Pull request merged**